specs/notation-cli.md

```diff
@@ -127,6 +129,32 @@ OPTIONS:
    --help, -h                  show help (default: false)
 ```
 
+## login
+
+```console
+notation login --help
+NAME:
+   notation login - Provides credentials for authenticated registry operations
+
+USAGE:
+   notation login [options] [server]
+
+OPTIONS:
+   --username value, -u value    Username for registry operations
+   --password value, -p value    Password for registry operations
```

Comment:
How about environment variables associated with `-u` and `-p` like `NOTATION_USERNAME` and `NOTATION_PASSWORD`?

Reply:
```suggestion
   --username value, -u value    Username for registry authentication
   --password value, -p value    Password for registry authentication
```

Reply:
Should we also support `--password-stdin` like [docker cli](https://docs.docker.com/engine/reference/commandline/login/#provide-a-password-using-stdin) does?